pkg/controller/node/nodecontroller.go

```diff
@@ -343,6 +386,17 @@ func forcefullyDeletePod(c client.Interface, pod *api.Pod) {
 	}
 }
 
+// releaseCIDR does translating CIDR back to CIDR index and insert this index
+// back to availableCIDRs set
+func (nc *NodeController) releaseCIDR(freeCIDR string) {
+	CIDRsArray := strings.Split(freeCIDR, ".")
```

Comment:
This is designed for /24, if we change it to /26 or make it configurable, we need to change this function a little bit. It will not be a big change if we change /24 to /x. This has the same TODO as TODO in generateCIDRs: // TODO: Make the CIDRs configurable.